### PR TITLE
[AMORO-2637] Enable return back to origin uri when relogin

### LINKED ISSUE #2637: [Improvement]: Enable returnUrl parameter when redirecting to login page
### Search before asking

- [X] I have searched in the [issues](https://github.com/NetEase/amoro/issues?q=is%3Aissue) and found no similar issues.

### What would you like to be improved?

I think it is worth enabling returnUrl parameter. If sharing amoro url with others, he will act two steps:
   1) paste the url and login in successfully, 
   2) paste it again and access.

### How should we improve?

_No response_

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Subtasks

_No response_

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by tcodehuber**
## Why are the changes needed?

Close #2637 .

## Brief change log

-

## How was this patch tested?

- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [ ] Run test locally before making a pull request

## Documentation

- Does this pull request introduce a new feature? (yes / no)
- If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)

### CODE REVIEW

File: ams/server/src/main/java/com/netease/arctic/server/dashboard/DashboardServer.java [hunk 1]
```diff
@@ -394,8 +394,8 @@ public void preHandleRequest(Context ctx) {
           ctx.queryParam("signature"),
           ctx.queryParamMap());
     } else if (needLoginCheck(uriPath)) {
+      LOG.error("---+++++: " + ctx.path());
```
Comment (wangtaohz) [inline ams/server/src/main/java/com/netease/arctic/server/dashboard/DashboardServer.java:397]: Is this redundant? It will introduce too many error logs.

Comment (tcodehuber) [reply ams/server/src/main/java/com/netease/arctic/server/dashboard/DashboardServer.java]: > Is this redundant? It will introduce too many error logs.

Oh, it's really my fault. Thanks for your comment. I have removed the unused code, plz check it again.

**Review by baiyangtx — COMMENTED** [review]
If the dashboard module's codes are modified, please recompile the project frontend

**Comment by baiyangtx** [comment]
@minteliuwm PTAL

**Comment by tcodehuber** [comment]
> If the dashboard module's codes are modified, please recompile the project frontend

Now the front end can automatically submit the built files through the Maven plugin.

**Review by minteliuwm — APPROVED** [review]
LGTM

**Review by zhoujinsong — APPROVED** [review]
LGTM.
